### PR TITLE
feat: export out of root folder 

### PR DESCRIPTION
**Pull request by leamendoza97**
# What? 
Change the url makers to add the pathname instead of only host.

# Why? 
This way allows serve the export from any path instead of only root.

### CODE REVIEW

File: static/export.html [hunk 1]
```diff
@@ -566,7 +570,7 @@
           return `https://renderer-artifacts.decentraland.org/branch/${qs.get('renderer-branch')}`
         }
 
-        return new URL(`${location.origin}${location.pathname}unity-renderer`).toString()
+        return new URL(rootURL() + unity-renderer).toString()
```
Comment (menduz) [inline static/export.html]:
```suggestion
        return new URL(rootURL() + 'unity-renderer').toString()
```

File: packages/config/index.ts [hunk 2]
```diff
@@ -141,13 +141,17 @@ export const FORCE_SEND_MESSAGE = location.search.includes('FORCE_SEND_MESSAGE')
...
 
-export const previewPinCatalyst = () => {
+export const rootURLPreviewMode = ({ endsWithSlash }: { endsWithSlash: boolean } = { endsWithSlash: false }) => {
   const url = `${location.origin}${location.pathname}`
-  return url.endsWith('/') ? url.substr(0, url.length-1) : url
+  if (url.endsWith('/')) {
+    return endsWithSlash ? url : url.replace(/$\//, '')
+  } else {
+    return !endsWithSlash ? url : `${url}/`
+  }
```
Comment (menduz) [inline packages/config/index.ts]: I rather create two functions with different names for this, the logic was very hard to follow without context

Comment (leamendoza97) [reply packages/config/index.ts]: Ok!, I think all this logic will need to move to some module in the future with a location abstraction.

**Review by nearnshaw — APPROVED** [review]
